gcp/appengine/frontend3/src/templates/list.html

```diff
@@ -169,5 +169,10 @@ <h1 class="title">Vulnerability Library</h1>
 <script>
   import { MDCDataTable } from '@material/data-table';
   const dataTable = new MDCDataTable(document.querySelector('.mdc-data-table'));
+  // Remove extra spaces from search query
+  document.getElementById('search-form').addEventListener('submit', function (e) {
```

Comment:
Instead of doing this in javascript, we'd prefer this to be handled server-side instead. See https://github.com/google/osv.dev/blob/master/gcp/appengine/frontend_handlers.py